tox.ini

```diff
@@ -4,18 +4,20 @@ envlist =
     build_docs
     codestyle
     py38-conda
-isolated_build = true
 requires =
     setuptools >= 30.3.0
     pip >= 19.3.1
     tox-pypi-filter >= 0.10
+    extension_helpers
+    numpy
```

Comment:
This is probably bad from a perspective of verifying our sdists install into a clean env on other setups other than conda

Reply:
Undone.